.circleci/config.yml

```diff
@@ -23,8 +23,7 @@ jobs:
     steps:
       - log_date
       - checkout
-      - restore_cache:
-          key: go-mod-v1-{{ checksum "go.sum" }}
+      - run: go mod download
```

Comment:
Is this supposed to be faster than not having this step at all?

Reply:
Good question. No, it won't make the build any faster. I suppose all it will do is make download errors stand out. With that in mind, think we should keep it?

Reply:
I think this change's benefits outweigh what was there before, so my vote is to keep it. I know this is a drive-by review so I'll just let my vote be a soft +1.